libsolidity/parsing/Parser.cpp

```diff
@@ -380,16 +398,47 @@ ASTPointer<ContractDefinition> Parser::parseContractDefinition()
 	std::vector<ASTPointer<InheritanceSpecifier>> baseContracts;
 	std::vector<ASTPointer<ASTNode>> subNodes;
 	std::pair<ContractKind, bool> contractKind{};
+	ASTPointer<Expression> storageBaseLocationExpression;
 	documentation = parseStructuredDocumentation();
 	contractKind = parseContractKind();
 	std::tie(name, nameLocation) = expectIdentifierWithLocation();
-	if (m_scanner->currentToken() == Token::Is)
-		do
+	while (true)
+	{
+		if (m_scanner->currentToken() == Token::Is)
 		{
-			advance();
-			baseContracts.push_back(parseInheritanceSpecifier());
+			if (baseContracts.size())
+				m_errorReporter.parserError(
+					6668_error,
+					m_scanner->currentLocation(),
+					SecondarySourceLocation().append("First inheritance definition is here", baseContracts[0]->location()),
```

Comment:
Is this actually output anywhere? I can't see the secondary source location messages anywhere in the tests (not just for this particular one, but all of them in general).

Reply:
They appear in the CLI, not sure if they do in the JSON output.

Reply:
Hmm, weird, I did a bit of grepping around and didn't see any occurrences in the tests, but I might be mistaken.

Reply:
Yeah, I hadn't added any cmdline tests, so the secondary source location/error message was not showing.
I added two more tests covering this case (inheritance) and the repetead base location definition.

Reply:
Agreed, but I think we should have *some* way to test those. Untested functionality tends to be broken.

I think that a good middle-ground would be a simple test setting like `showSecondaryLocations: true` (`false` by default), that you could enable to show them in those syntax tests where you care about them.